tmt.spec

```diff
@@ -147,6 +147,50 @@ install -pm 644 %{name}/steps/provision/mrack/mrack* %{buildroot}/etc/%{name}/
 %config(noreplace) %{_sysconfdir}/%{name}/mrack*
 
 %changelog
+* Mon Nov 06 2023 Lukáš Zachar <lzachar@redhat.com> - 1.29.0
+- Refactor generating of stories and lint check docs
```

Comment:
Last commit from `main` seems to be missing:

    Update and polish hardware requirement docs